feature-libs/organization/user-registration/components/form/user-registration-form.service.ts

```diff
@@ -163,12 +165,14 @@ export class UserRegistrationFormService {
   /**
    * Redirects the user back to the login page.
    *
-   * This only happens in case of the `ResourceOwnerPasswordFlow` OAuth flow.
+   * This only happens in case of the `ResourceOwnerPasswordFlow` OAuth flow and authorizationCodeFlowByDefault.
+   *
    */
   protected redirectToLogin(): void {
     if (
       this.authConfigService.getOAuthFlow() ===
-      OAuthFlow.ResourceOwnerPasswordFlow
+        OAuthFlow.ResourceOwnerPasswordFlow ||
+      this.featureConfigService.isEnabled('authorizationCodeFlowByDefault')
```

Comment:
Is it necessary here? if it was not changed here (unless in epic we introduce that change?)

Reply:
Ah ok, missed that its not a part of fixing hardcoded routes, but also neccesary change logic for flow